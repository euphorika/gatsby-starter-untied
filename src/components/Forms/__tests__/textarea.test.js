import React from 'react'
import renderer from 'react-test-renderer'
import Textarea from '../textarea'

describe('Textarea', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Textarea name="Name" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
