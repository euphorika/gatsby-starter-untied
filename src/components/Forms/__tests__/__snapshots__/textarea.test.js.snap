// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Textarea renders correctly 1`] = `
<div
  className="elementContainer"
>
  <textarea
    className="element"
    name="Name"
  />
  <span
    className="afterFormElement"
  />
</div>
`;

exports[`Textarea renders required correctly 1`] = `
<div
  className="elementContainer"
>
  <textarea
    className="element"
    name="Name"
    required={true}
  />
  <span
    className="afterFormElement"
  />
</div>
`;
